compactor/compactor.go

```diff
@@ -32,8 +31,24 @@ var (
 const (
 	checkCompactionInterval   = 5 * time.Minute
 	executeCompactionInterval = time.Hour
+
+	ModePeriodic   = "period"
```

Comment:
Got it.
Should I rename the `Revisional` to `Revision` and `ModeRevisonal = "revision"` to `ModeRevison = "revision"`? mixing "revisional" and "revision" could be confusing.